README.md

```diff
@@ -6,6 +6,9 @@
 [![eslint github action](https://img.shields.io/github/workflow/status/redhat-developer/vscode-xml/lint?label=eslint&logo=eslint&style=for-the-badge)](https://github.com/redhat-developer/vscode-xml/actions/workflows/lint.yaml?query=branch%3Amain)
 
 ### [**NO LONGER REQUIRES JAVA!**](#Requirements) *since v0.15.0*
+### [**RelaxNG support!**](https://github.com/redhat-developer/vscode-xml/blob/main/docs/RelaxNGSupport) *since v0.22.0*
+
+![RelaxNG support](docs/images/RelaxNG/RelaxNGSupport.gif)
```

Comment:
The demo takes 1.7 MB, is it annoying?

Reply:
It would be nice to reduce the size of this, it's large in comparison to the lemminx uber (9 MB) but not in comparison to the binary (46 MB)

Reply:
Ok now it takes 677 KB